### DISCUSSION TITLE
export point data

### DISCUSSION THREAD

**Discussion by alessiofumagalli** [pmgbergen/porepy#678] [Ideas]
Dear all, I'm interested to export point data. have you already planned this task? Are you interested to have this addition to pp?

**Comment by IvarStefansson** [comment]
I am not aware of any such plans. @keileg might have an opinion on this, while @jwboth has recently overhauled the exporter. I suggest we wait for them to return from their holidays.

**Reply by keileg** [reply]
I think it will be a nice contribution, provided it can be accommodated within the new Exporter framework. @jwboth, what do you think?

**Reply by jwboth** [reply]
My feeling is that it should be possible. So far the exporter is not exporting any 0d grids. So there is currently no conflict of interest and it should be possible to integrate the export of point data by extending the exporter (in `export_grid` for instance and at analogous places).

**Reply by jwboth** [reply]
I can have a look at it (this item would most likely not end up on the top of my todo list this week though).
